frontend/.gitignore

```diff
@@ -35,7 +35,7 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 
-.env*
```

Comment:
Hi, this is important because there can be other important .envs, if you want to include one or more .env in the commit just use !name